src/meta/app_balance_policy.h

```diff
@@ -25,20 +25,15 @@ class app_balance_policy : public load_balance_policy
 {
 public:
     app_balance_policy(meta_service *svc);
```

Comment:
app_balance_policy(meta_service *svc) explicit;

Reply:
OK, let's refactor it in another patch.